modules/json/src/json-converter.js

```diff
@@ -125,6 +129,24 @@ function convertClassInstance(json, configuration) {
   return instantiateClass(type, props, configuration);
 }
 
+// Plain JS object, embed functions.
+function convertFunctionObject(json, configuration) {
+  if (configuration.functions && configuration.convertFunction) {
+    const {functionKey} = configuration;
+
+    const targetFunction = json[functionKey];
+    const dataProp = json.prop;
```

Comment:
`prop` is an overloaded term in deck.gl. If this only refers to a path in a data row, use `key`. If this is an expression, use `value`.